scalameta/src/main/scala/scala/meta/internal/parsers/Parsers.scala

```diff
@@ -2111,14 +2149,16 @@ private[meta] class Parser(val input: Input)(implicit val dialect: Dialect) { pa
     def appendMod(mods: List[Mod], mod: Mod): List[Mod] = {
       def validate() = {
         if (isLocal && !mod.tokens.head.is[LocalModifier]) syntaxError("illegal modifier for a local definition", at = mod)
-        mods.foreach(m => if (m.productPrefix == mod.productPrefix) syntaxError("repeated modifier", at = mod))
+        if (!mod.isInstanceOf[Mod.Quasi]) mods.foreach(m => if (m.productPrefix == mod.productPrefix) syntaxError("repeated modifier", at = mod))
         if (mod.hasAccessBoundary) mods.filter(_.hasAccessBoundary).foreach(mod => syntaxError("duplicate access qualifier", at = mod))
       }
       validate()
       mods :+ mod
     }
+    def continueLoop = ahead(token.is[`:`] || token.is[`=`] || token.is[EOF])
```

Comment:
This is probably the only thing in this pull request that looked seriously spooky to me. I'd prefer you document it by saying something like "the only things that can come after $mod or $mods are either keywords or $pname/$tpname; the former is easy, but in the case of the latter, we need to take care to not hastily parse those names as modifiers".